proto/buf/alpha/registry/v1alpha1/repository_commit.proto

```diff
@@ -40,6 +40,8 @@ message RepositoryCommit {
   string author = 7;
   // The tags associated with this commit
   repeated buf.alpha.registry.v1alpha1.RepositoryTag tags = 8;
+  // The commit's draft name, if it is a draft commit.
+  optional string draft_name = 9;
```

Comment:
We're not avid proto3 optional users, but this seemed most appropriate. As draft names are always non empty, we could remove the `optional` here and interpret empty as not a draft commit.

Reply:
I would just remove the optional and add a comment that empty is no draft.

Reply:
For languages ​​with nullable types, proto3 optionals usually provide a better experience than comparing against a zero value. I welcome exposure to this feature in our code base, but am okay with both ways.

Reply:
I really dislike proto3 optionals - is there any reason to disambiguate between unset and empty here? Is it possible to have a valid, empty draft name? If not, there is likely no use for this.

Reply:
Removed `optional` to match the rest of the api.